src/org/omegat/gui/main/IMainWindow.java

```diff
@@ -204,4 +204,9 @@ public interface IMainWindow {
      * Retrieve main docking desktop.
      */
     DockingDesktop getDesktop();
+
+    /**
+     * Restores the main window layout to the default values.
+     */
+    default void resetDesktopLayout() {}
```

Comment:
We add it as `default`  which keeps backward compatibility for plugins compiled with previous versions of OmegaT.